package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "design-system-react",
-	"version": "0.10.45",
+	"version": "0.10.48",
```

Comment:
This change here happened right away after I forked and ran `npm install`. Seems like this version does line up with the current version.